library/src/main/java/com/google/android/exoplayer/text/webvtt/WebvttCue.java

```diff
@@ -36,7 +36,7 @@ public WebvttCue(long startTime, long endTime, CharSequence text) {
   }
 
   public WebvttCue(long startTime, long endTime, CharSequence text, int line, int position,
-      Alignment alignment, int size) {
+                   Alignment alignment, int size) {
```

Comment:
Fully revert this file to avoid git history pollution :)? (see also rm'd newline at bottom).